### ISSUE TITLE
Sideloaded 2.1.08 doesn't appear in apps list on latest gen Fire TV Stick with voice

### ISSUE THREAD

**Issue by 8none1** [oakesville/mythling#89]
I've sideloaded 2.1.08 on to both of my TV Sticks.  One is the newest version with the voice remote as standard and one is the original version with the non-voice remote.

Mythling appears in the apps list from the home screen on the older device, but not on the newer one.

I can still launch it by going in to the installed apps setting and it works fine, just that I can't launch it from the main home screen.

I wonder if the manifest file has changed in the newest devices.

**Comment by oakesville** [comment]
On the new Fire TV Stick, try installing standard Mythling from Amazon instead of sideloading.  Then follow this by sideloading the updated Mythling.
 - Don

**Comment by 8none1** [comment]
Gave that a go today, and discovered that Amazon think that Mythling is not compatible with the new TV Stick, so that might explain it.

http://imgur.com/a/Sf6ym

FWIW - it works just fine.

**Comment by oakesville** [comment]
I went ahead and submitted 2.1.08 for release on FireTV.  It should be available in the next day or two.  You'll probably need to uninstall the sideloaded version to install from Amazon.  Since Live App Testing was a bust, please let me know if you observe any issues with Mythling 2.1.08.

Thanks,
 - Don

**Comment by 8none1** [comment]
The new version is now in the store, but it's reporting as not being compatible with either of my FireTV sticks now:

![image](https://cloud.githubusercontent.com/assets/6552931/25170813/bac343ac-24e3-11e7-812d-2fa61f5ae0ae.png)

**Comment by oakesville** [comment]
Mythling always shows as incompatible for my FireTV on the Amazon site.  However, if I install per these instructions it seems to work:
https://github.com/oakesville/mythling/wiki/Mythling-On-Fire-TV#1-from-amazon

Once upon a time I opened a support issue with Amazon about this and they told me that this was normal since the browser shows compatibility for mobile devices and not Fire TV.  Hopefully you can still install 2.1.08.  If not, please let me know and I'll uninstall myself and try to reproduce.

Thanks,
 - Don

**Comment by 8none1** [comment]
I gave that a go, but no luck.

I've uninstalled the side-loaded version and rebooted for good measure.  "Bought" the app via Amazon website and did the sync content per the instructions - but it never shows up as available to install from the cloud.

I checked the manifest file, but I couldn't see anything wrong there (e.g. not requiring a touch screen is there).  I also uploaded the apk to the Amazon App Testing Service.  It did check it on Fire TV sticks and boxes, and found no errors.  So as far as I can tell the app tester doesn't think it's a problem.  Link: https://developer.amazon.com/app-testing-service

**Comment by 8none1** [comment]
In case it's relevant, My Fire TV Sticks are running Fire OS 5.2.4.1 which according to this page:  https://developer.amazon.com/public/support/submitting-your-app/tech-docs/targeting-amazon-devices-with-your-android-manifest
points to a spreadsheet which suggests that Fire OS 5 supports v. 22 of the SDK.

**Comment by oakesville** [comment]
Thanks for those details.  I finally got to the bottom of the compatibility issue.  It seems that Amazon now requires some additional mandatory promotional artwork.  Unfortunately this requires publishing a new build.  I'll make 2.1.10 available within the next day or two.

**Comment by oakesville** [comment]
Okay, 2.1.10 is live on Amazon.  For me Mythling still shows as incompatible with my FireTV device on amazon.com.  However, I completely uninstalled, resynced my apps, and was able to install again.  Please tell me you're able to install now.

**Comment by 8none1** [comment]
Maybe I'm doing something stupid (likely) but it's still not working for me.

Steps I followed:

1. Uninstall side-loaded version.
2. Bought the app via Amazon.co.uk & .com just in case.
3. Resynced the apps
4. Go to the "Your Apps & Games" on the home screen - and nothing appears.

Next step is to factory reset the stick and try again.

For testing, I downloaded the apk with a Chrome extension and side-loaded it.  It works just fine (but doesn't appear in my list of apps, but that's probably because it's side-loaded).

I'll reset the stick and try again and report back.
